test/nbody_thermostat_test.jl

```diff
@@ -0,0 +1,39 @@
+println("====  Andersen thermostat testing    ====")
```

Comment:
I see, this is all in SI units. We can handle that in another PR.

Reply:
well, yes, I haven't changed these files for the OpenMM of Gromaks units yet